mergify_engine/actions/merge.py

```diff
@@ -41,14 +41,9 @@ class MergeAction(merge_base.MergeBaseAction):
         voluptuous.Required("rebase_fallback", default="merge"): voluptuous.Any(
             "merge", "squash", None
         ),
-        voluptuous.Required("strict", default=False): voluptuous.Any(
-            bool,
-            voluptuous.All("smart", voluptuous.Coerce(lambda _: "smart+ordered")),
-            voluptuous.All(
-                "smart+fastpath", voluptuous.Coerce(lambda _: "smart+fasttrack")
-            ),
-            "smart+fasttrack",
-            "smart+ordered",
+        voluptuous.Required("strict", default=False): voluptuous.All(
```

Comment:
```suggestion
        voluptuous.Required("strict", default=False): voluptuous.All(
```
the default is not part of the `Enum`